site/ts/instructor-submission.ts

```diff
@@ -40,7 +40,7 @@ function init(){
     const qrPrefixInput = document.getElementById('qr_prefix') as HTMLInputElement;
     const qrSuffixInput = document.getElementById('qr_suffix') as HTMLInputElement;
     const useQRCheckBox = document.getElementById('use-qr') as HTMLInputElement;
-    const useScanIdsCheckBox = document.getElementById('use-ocr') as HTMLInputElement;
+    const useScanIdsCheckBox = document.getElementById('use-ocr') as HTMLInputElement | null;
```

Comment:
This change is essentially the same as removing the type assertion altogether and just having:
```
    const useScanIdsCheckBox = document.getElementById('use-ocr');
```

Reply:
document.getElementById provides `HTMLElement | null` - HTMLInputElement provides some additional properties such as the `checked` property so just having `document.getElementById('use-ocr')` throws an error with the checker.

Reply:
Fair